.github/workflows/build-chart-jfrog.yaml

```diff
@@ -15,17 +15,62 @@ on:
         description: 'Name of the jfrog repo to deploy the chart to'
         required: true
         default: 'ecosystem-helm-dev-local'
-      build_name:
-        description: 'Name of the build to create'
-        required: true
-        default: 'aerospike-vector-search-helm'
-      build_number: 
+      chart_version: 
         description: 'Build number to use for the build metadata'
         required: true
         default: '0.7.0'
+env:
+    JF_PROJECT: ${{ github.event.inputs.jfrog_project }}
+    JF_REPO: ${{ github.event.inputs.repo_name }}
+    CHART_VERSION: ${{ github.event.inputs.chart_version }}
+    CHART_NAME: ${{ github.event.inputs.chart_name }}
+
 jobs:
   build-chart:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
+      - name: Checkout current repository
+        uses: actions/checkout@v3
+
+    # You can sign helm charts but not doing this for now
+    #   - name: setup GPG
+    #     uses: aerospike/shared-workflows/devops/setup-gpg@main
+    #     with:
+    #       gpg-private-key: ${{ secrets.GPG_SECRET_KEY }}
+    #       gpg-public-key: ${{ secrets.GPG_PUBLIC_KEY }}
+    #       gpg-key-pass: ${{ secrets.GPG_PASS }}
+    #       gpg-key-name: "aerospike-inc"
+          
+      - name: setup jfrog
+        uses: jfrog/setup-jfrog-cli@v4
+        env:
+          JF_URL: https://aerospike.jfrog.io
+          JF_ACCESS_TOKEN: ${{ secrets.JF_ECO_CONTRIB_TOKEN }}
+          JF_PROJECT: ${{ github.event.inputs.jfrog_project }}
+
+        
+      - name: "Deploy helm to JFrog"
+        run: |
+            helm package $CHART_NAME
```

Comment:
Is CHART_VERSION used in the name of the artifact that this command produces? Otherwise I don't see how the next command works.

Reply:
Oh, I think I get it, --build-number="${{env.CHART_VERSION}} adds it

Reply:
yep